client/assets/javascripts/stores/CommentStore.js

```diff
@@ -1,7 +1,10 @@
-import { createStore, applyMiddleware } from 'redux';
-import commentReducer from '../reducers/commentReducer';
+import { createStore, applyMiddleware, combineReducers } from 'redux';
+import thunk from 'redux-thunk';
+import reducers from '../reducers';
+
+// Smth wrong with this one, off for now
 import loggerMiddleware from '../middleware/loggerMiddleware';
```

Comment:
Yes -- let's see if we can get this one working afterwards, maybe a separate PR.